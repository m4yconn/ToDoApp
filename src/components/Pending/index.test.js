import React from 'react';
import renderer from 'react-test-renderer';
import Pending from '../Pending'

test("Snapshot correto", () => {
  const tree = renderer.create(<Pending/>).toJSON();
  expect(tree).toMatchSnapshot();
});