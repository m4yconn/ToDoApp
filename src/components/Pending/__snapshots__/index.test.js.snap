// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot correto 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "backgroundColor": "#dcdcdc",
      "borderColor": "#8b008b",
      "borderRadius": 5,
      "borderWidth": 2,
      "flexDirection": "center",
      "marginBottom": 10,
      "padding": 5,
    }
  }
>
  <Text
    style={
      {
        "fontFamily": "courier",
        "fontSize": 20,
        "fontWeight": "bold",
        "marginLeft": 20,
        "textDecorationLine": "line-through",
      }
    }
  />
</View>
`;
